lib/Sema/CSApply.cpp

```diff
@@ -6818,8 +6818,9 @@ ExprRewriter::finishApplyDynamicCallable(const Solution &solution,
       cs.setType(labelExpr, keyType);
       handleStringLiteralExpr(cast<LiteralExpr>(labelExpr));
 
-      Expr *pair =
-        TupleExpr::createImplicit(ctx, { labelExpr, arg->getElement(i) }, {});
+      Expr *valueExpr = coerceToType(arg->getElement(i), valueType, loc);
+      if (!valueExpr) return nullptr;
```

Comment:
Nit: clang-format doesn't put return on the same line like that. But to be fair there are other cases like that here too, we should definitely be more consistent in style